bot/extensions/custom_roles/commands.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+
+import discord
+from discord import app_commands
+from discord.ext import commands
+from discord.utils import escape_markdown, format_dt
+
+from bot import core
+from bot.models.custom_roles import CustomRole
+
+
+class CustomRoles(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    def role_embed(self, heading: str, user: discord.Member, role: discord.Role):
+        embed = discord.Embed(
+            description=f"### {heading} {user.mention}\n\n**Name**\n{role.name}\n**Color**\n{role.color}"
+            f"\n**Created**\n {format_dt(role.created_at)}",
+            color=role.color,
+            timestamp=datetime.datetime.utcnow(),
+        )
+        embed.set_footer(text=user.name)
+        embed.set_thumbnail(url=user.avatar)
+        return embed
```

Comment:
Perhaps I need to see these embeds but I feel like we don't need this function.
I also explained before development how we wanted the embed to look 👀, for example this is missing a marker for updated fields